examples/Training/python/Json_Api/Login.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#
+# Copyright (C) 2016 University of Dundee & Open Microscopy Environment.
+#                    All Rights Reserved.
+# Use is subject to license terms supplied in LICENSE.txt
+#
+
+import requests
+
+session = requests.Session()
+
+# Start by getting supported versions from the base url...
+r = session.get('http://localhost:4080/api/')
```

Comment:
Great you remembered about training examples. Could we not put `localhost:4080`. All the properties are in https://github.com/openmicroscopy/openmicroscopy/blob/develop/examples/Training/python/Parse_OMERO_Properties.py. Could you just add something like `OMERO_WEB_HOST = localhost`